docs/rel_0.5.0.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Version 0.5.0
+keywords: apache, hudi, release, 0.5.0, data lake, upsert,
+sidebar: mydoc_sidebar
+permalink: rel_0.5.0.html
+toc: false
+summary: "Release 0.5.0 Page"
+---
+
+Apache Hudi (incubating) Latest Stable Release is : 0.5.0-incubating
+
+## Download
+  * Source Release : [Apache Hudi(incubating) 0.5.0-incubating Source Release](https://www.apache.org/dist/incubator/hudi/0.5.0-incubating/hudi-0.5.0-incubating.src.tgz) ([asc](https://www.apache.org/dist/incubator/hudi/0.5.0-incubating/hudi-0.5.0-incubating.src.tgz.asc), [sha512](https://www.apache.org/dist/incubator/hudi/0.5.0-incubating/hudi-0.5.0-incubating.src.tgz.sha512))
```

Comment:
404 not found, will these links available after release?

Reply:
Thanks @leesf  for checking. Yes, those links should appear soon.